internal/web/context.go

```diff
@@ -0,0 +1,22 @@
+package web
+
+import (
+	"github.com/labstack/echo/v4"
+	"go.uber.org/zap"
+)
+
```

Comment:
The AppContext embeds echo.Context, which can lead to confusion due to method name shadowing. Instead of using embedding, consider defining AppContext to have a Context field (e.g., `Context echo.Context`), so it’s clear where methods are coming from. This change increases code clarity regarding which context is being used.